src/quic/endpoint.cc

```diff
@@ -708,33 +748,18 @@ void Endpoint::Listen(const Session::Options& options) {
 BaseObjectPtr<Session> Endpoint::Connect(
     const SocketAddress& remote_address,
     const Session::Options& options,
-    std::optional<SessionTicket> sessionTicket) {
-  // TODO(@jasnell): Implement as part of Session...
+    std::optional<SessionTicket> session_ticket) {
```

Comment:
Why not pass the pointer instead of std::optional to avoid unnecessary copy?

Reply:
I'll tackle this in a follow on PR. I'm going to be revisiting the use of `std::optional` entirely in all this as after using it for a bit I'm generally not very happy with it.